lighthub/main.h

```diff
@@ -25,14 +25,14 @@ void Changed (int i, DeviceAddress addr, int val);
 void modbusIdle(void);
 void _handleHelp(int arg_cnt, char **args);
 void _kill(int arg_cnt, char **args);
-void parseConfig();
+void parseConfigFromParsedJSON();
```

Comment:
too sophisticated name

Reply:
may be parseConfigFromValidatedJSON? But this name shows that json is already processed and now config parsed using prepared data.